fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -309,16 +309,33 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 
 	struct Reader : IThreadPoolReceiver {
 		DB& db;
-
-		explicit Reader(DB& db) : db(db) {}
+		double readValueTimeout;
+		double readValuePrefixTimeout;
+		double readRangeTimeout;
+
+		explicit Reader(DB& db) : db(db) {
+			if (g_network->isSimulated()) {
+				// In simulation, increasing the read operation timeouts to 5 minutes, as some of the tests have
+				// very high load and single read thread cannot process all the load within the timeouts.
+				readValueTimeout = 5 * 60;
```

Comment:
@neethuhaneesha would this contribute to the timeout failure you are debugging? 
If a simulation scenario has a lot of reads that hit such high timeout threshold, simulation will run extremely slow and timeout, I think.